---
layout: post
title: Life Science Informatics 
date: 2022-12-04 12:35:20 +0900
# description: 
# img: 
# fig-caption: 
tags: [life science]
---

## 2022 December

### News
- [Nanome](https://nanome.ai/): VR for Molecular Design, a new kind of user interface.
- [VR分子モデリングシステムで研究員のインスピレーションを刺激。創薬化学研究部が追求する薬の候補の化合物デザインとは](https://note.chugai-pharm.co.jp/n/n5fc55cfb3fcc): Chugai Digital and Nanome
- [MI-6: 材料開発とデータサイエンスのHubとなり研究者のポテンシャルを解き放つ](https://mi-6.co.jp/): a Japanese startup for simulation of new material
- [Creative Quantum](https://www.creative-quantum.eu/en/home): Quantum Mechanical Simulations research at molecular level, a german software company
- [MolDesk](https://www.moldesk.com/): in silico drug design (molecular design) package software with myPresto and GROMACS.　インシリコ創薬による、ドラッグデザイン（薬剤分子設計） パッケージ ソフトウェア by [IMSBIO](https://www.imsbio.co.jp/), the company who commercializes myPresto.
- [AutoDock Vina on Qarnot Cloud – documentation](https://blog.qarnot.com/autodock-vina-on-qarnot-cloud-documentation/): AutoDock is a popular docking software for drug discovery. In 2022, it was put on cloud services provided by Qarnot.
- [Genesis](https://www.r-ccs.riken.jp/labs/cbrt/) GENESIS (GENeralized-Ensemble SImulation System) has been developed mainly by ***Sugita group in RIKEN***, (Computational Biophysics Research Team (R-CCS), Theoretical Molecular Science Laboratory (CPR), and Laboratory for Biomolecular Function Simulation (BDR)) since 2009. They use the [supercomputer Fugaku](https://en.wikipedia.org/wiki/Fugaku_(supercomputer)) as the computing infrastructure.
- [KEGG](https://www.kegg.jp/) is a database resource for understanding high-level functions and utilities of the biological system. KEGG is an academic project by [Kanehisa Labs](https://www.kanehisa.jp/en/). The lab has a [supercomputer system](https://www.kanehisa.jp/en/history.html). [data growth](https://www.kanehisa.jp/en/db_growth.html) of sequence and 3D structure databases. The commercial use of KEGG is provided by the company [Pathway Solutions](https://www.pathway.jp/)
- The National Center for Biotechnology Information (NCBI) Basic Local Alignment Search Tool[(BLAST)](https://github.com/ncbi/blast_plus_docs) finds regions of local similarity between sequences. Now it supports cloud computing by Google Cloud and AWS.
- [EMBL-EBI: Supporting small business](https://www.ebi.ac.uk/industry/smes/): Alzheimer’s Disease Data Initiative (ADDI): [presentation](https://www.ebi.ac.uk/industry/wp-content/uploads/public/B4BB/2022/12.-Matt-Clement-ADDI-B4BB-slides_Clement.pdf), [site]()
- [Virtual Screening of Novel Active Drug Compounds on AWS with Orion®](https://aws.amazon.com/blogs/hpc/virtual-screening-of-drug-compounds-with-orion/): by OpenEye
- [GROMACS in the Cloud: A Global Supercomputer to Speed Up Alchemical Drug Design](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00044): 4.2. Cost Comparison: Cloud vs On-Premises Cluster
- [nextflow](https://www.nextflow.io/blog.html), [seqeralabs](https://seqera.io/), [Seqera Labs raises €22 million in Series A funding led by Addition](https://seqera.io/news/seqera-labs-raises-eur-22-million-in-series-a-funding/), [Nextflow Summit 2022 - Evan Floden](https://www.youtube.com/watch?v=yJpN3fRSClA)

### Computing Modules
- [DiffDock: Diffusion Steps, Twists, and Turns for Molecular Docking](https://openreview.net/forum?id=SttOaKinOI), [code](https://anonymous.4open.science/r/DiffDock/README.md)
- [Molecular docking with Raccoon2 on clouds: extending desktop applications with cloud computing](https://ceur-ws.org/Vol-2363/paper2.pdf)
- [Raccoon2](https://autodock.scripps.edu/resources/raccoon2/): Raccoon2 is a graphical interface for preparing and analyzing AutoDock Vina virtual screenings. [AutoDock GPU on github](https://github.com/ccsb-scripps/AutoDock-GPU/blob/develop/host/src/main.cpp)
- [MINT: Motif Identifier for Nucleic acids Trajectory](http://mint.cent.uw.edu.pl/index.php?strona=MintInt)
- [Genesis](https://www.r-ccs.riken.jp/labs/cbrt/) 
- [rDock](http://rdock.sourceforge.net/) A Fast, Versatile and Open Source Program for Docking Ligands to Proteins and Nucleic Acids
- [rdchiral_cpp](https://gitlab.com/ljn917/rdchiral_cpp)
- [aizynthfinder](https://molecularai.github.io/aizynthfinder/cli.html): A fast, robust and flexible software for retrosynthetic planning
- [GROMACS Tutorials](http://www.mdtutorials.com/gmx/)
- [infinity-rust](https://github.com/utaal/infinity-rust): A rust wrapper for Infinity, a lightweight RDMA library for InfiniBand 
- [Welcome to ODU Research Computing Wiki!](https://wiki.hpc.odu.edu/)
- [Running GROMACS on AWS ParallelCluster](https://catalog.us-east-1.prod.workshops.aws/workshops/550e7f95-db06-487f-a14f-f80e000232f2/en-US)
- [Cloud-based molecular simulations for everyone](https://github.com/pablo-arantes/Making-it-rain)
- [Genome Analysis Toolkit](https://gatk.broadinstitute.org/hc/en-us) from Broad Institute
- [Illumina Open Source Repo](https://github.com/Illumina)
- [Clustal W & Cluster X](http://www.clustal.org/clustal2/)
- [OpenEye Software Integration Partners](https://www.eyesopen.com/software-integration-partners)

#### load ML models to C++ or Rust
- [Loading a TorchScript Model in C++](https://pytorch.org/tutorials/advanced/cpp_export.html)
- [CppFLow: Easily run TensorFlow models from C++](https://serizba.github.io/cppflow/#)
- [tch-rs](https://github.com/LaurentMazare/tch-rs)
- [Loading a trained HDF5 model into Rust to make predictions](https://stackoverflow.com/questions/68199756/loading-a-trained-hdf5-model-into-rust-to-make-predictions)
- [Salmon](https://combine-lab.github.io/salmon/getting_started/): Fast, accurate and bias-aware transcript quantification from RNA-seq data

#### A bioinformatics application
- [T-Coffee](https://tcoffee.org/Projects/tcoffee/index.html#DOWNLOAD), [another site](https://tcoffee.crg.eu/apps/tcoffee/index.html), for multiple alignment, [a video tutorial](https://www.youtube.com/watch?v=70VU_eYXPCg)
- Tutorial [BLAST and multiple sequence alignment (MSA) programs](https://viralzone.expasy.org/e_learning/alignments/content.html)
- [BMME 8053 Introduction to Bioinformatics: Online Tutorials](https://nsufl.libguides.com/bioinformatics/online)
